### PR TITLE
Feature/search patient

### PR DESCRIPTION
**Pull request by JelenaPopov**

### CODE REVIEW

File: Backend/src/main/java/ftn/tim16/ClinicalCentreSystem/dto/PatientWithIdDTO.java [hunk 1]
```diff
@@ -0,0 +1,122 @@
+package ftn.tim16.ClinicalCentreSystem.dto;
+
+import ftn.tim16.ClinicalCentreSystem.model.Patient;
+
+public class PatientWithIdDTO {
```
Comment (NikolaBrodic) [inline Backend/src/main/java/ftn/tim16/ClinicalCentreSystem/dto/PatientWithIdDTO.java]: Please remove unnecessary blank lines.

File: Frontend/src/app/components/list-of-examination-types/list-of-examination-types.component.ts [hunk 2]
```diff
@@ -63,6 +63,7 @@ export class ListOfExaminationTypesComponent implements OnInit {
 
   }
   sortEvent() {
+    console.log("sort");
```
Comment (NikolaBrodic) [inline Frontend/src/app/components/list-of-examination-types/list-of-examination-types.component.ts]: You should remove all console logs before commiting.

File: Backend/src/main/java/ftn/tim16/ClinicalCentreSystem/dto/PatientWithIdDTO.java (hunk 1 above)
Comment (JelenaPopov) [reply Backend/src/main/java/ftn/tim16/ClinicalCentreSystem/dto/PatientWithIdDTO.java]: Sorry, I will remove all unnecessary blank lines.

File: Frontend/src/app/components/list-of-examination-types/list-of-examination-types.component.ts (hunk 2 above)
Comment (JelenaPopov) [reply Frontend/src/app/components/list-of-examination-types/list-of-examination-types.component.ts]: Sorry, I forgot to delete that console.log.

**Pull request merged**